### DISCUSSION TITLE
【新闻】著名盗洞客“绳匠”神秘失踪，据悉跌入神秘空间

### DISCUSSION THREAD

**Discussion by WISNFB** [share121/inter-knot#3449] [其他]
空洞2046年8月33日，一位不愿透露名字的“fairy”先生向治安局报告，称其服务对象失踪，不知去向。经过治安局的调查，可能卡入了[后 室](https://backrooms-wiki-cn.wikidot.com/)，目前治安局正在组织人力尝试营救，希望市民们保持冷静。![image](https://github.com/user-attachments/assets/aa8a3495-750c-4b7f-83ee-cb4c4f67dc05)

**Comment by skysnown** [comment]
不是空洞吗？还是后室里面有空洞？
